### PR TITLE
Issue#2908

### PR DESCRIPTION
**Pull request by j0rt3g4**
Hello @RSEEGATZ, 
Sadly the answer is **no**. This is an Online feature -> For exchange online mailboxes.

Added this information as note on this PR (for issue #2908

### CODE REVIEW

**Comment by j0rt3g4** [comment]
Copy review @nenonix @JohanFreelancer9 is required on this one guys. Thank you in advance!

File: Teams/teams-scoped-directory-search.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ Microsoft Teams scoped directory search allows organizations to create virtual b
 
 Microsoft Teams lets organizations provide custom views of the directory to their users. Microsoft Teams uses [Exchange address book policies](https://docs.microsoft.com/exchange/address-books/address-book-policies/address-book-policies) to support these custom views. Once the policies are enabled, the results returned by searches for other users (for example, to initiate a chat or to add members to a team) will be scoped according to the configured policies. Users will not be able to search or discover teams when scoped search is in effect. 
 
+> [!NOTE]
+> On Exchange hybrid enviroments, this feature will only work correctly for exchange online mailboxes, not for on-premises mailboxes.
```
Comment (JohanFreelancer9) [inline Teams/teams-scoped-directory-search.md]:
```suggestion
> In Exchange hybrid environments, this feature will only work correctly with Exchange Online mailboxes, and not with on-premises mailboxes.
```

**Comment by j0rt3g4** [comment]
(Thank you @JohanFreelancer9 )

File: Teams/teams-scoped-directory-search.md [hunk 2]
```diff
@@ -27,6 +27,10 @@ Microsoft Teams scoped directory search allows organizations to create virtual b
 
 Microsoft Teams lets organizations provide custom views of the directory to their users. Microsoft Teams uses [Exchange address book policies](https://docs.microsoft.com/exchange/address-books/address-book-policies/address-book-policies) to support these custom views. Once the policies are enabled, the results returned by searches for other users (for example, to initiate a chat or to add members to a team) will be scoped according to the configured policies. Users will not be able to search or discover teams when scoped search is in effect. 
 
+> [!NOTE]
+> In Exchange hybrid environments, this feature will only work correctly with Exchange Online mailboxes, and not with on-premises mailboxes.
+
+
```
Comment (nenonix) [inline Teams/teams-scoped-directory-search.md]:
```suggestion
```

File: Teams/teams-scoped-directory-search.md [hunk 3]
```diff
@@ -27,6 +27,10 @@ Microsoft Teams scoped directory search allows organizations to create virtual b
 
 Microsoft Teams lets organizations provide custom views of the directory to their users. Microsoft Teams uses [Exchange address book policies](https://docs.microsoft.com/exchange/address-books/address-book-policies/address-book-policies) to support these custom views. Once the policies are enabled, the results returned by searches for other users (for example, to initiate a chat or to add members to a team) will be scoped according to the configured policies. Users will not be able to search or discover teams when scoped search is in effect. 
 
+> [!NOTE]
+> In Exchange hybrid environments, this feature will only work correctly with Exchange Online mailboxes, and not with on-premises mailboxes.
```
Comment (nenonix) [inline Teams/teams-scoped-directory-search.md]:
```suggestion
> In Exchange hybrid environments, this feature only works with Exchange Online mailboxes, and not with on-premises mailboxes.
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by LolaJacobsen** [comment]
@vinbelms @somakbhattacharyya - please review and LMK whether it's OK to merge this PR. Look at the last entry (directly above).

**Comment by scanum** [comment]
@vinbelms @somakbhattacharyya Following up on this PR. Any updates? Thanks!

**Pull request merged**